relay-event-normalization/src/event.rs

```diff
@@ -160,57 +129,41 @@ impl<'a> Default for NormalizationConfig<'a> {
     }
 }
 
-/// Normalizes an event, rejecting it if necessary.
+/// Normalizes an event.
 ///
 /// Normalization consists of applying a series of transformations on the event
 /// payload based on the given configuration.
 ///
 /// The returned [`ProcessingResult`] indicates whether the passed event should
 /// be ingested or dropped.
-pub fn normalize_event(
-    event: &mut Annotated<Event>,
-    config: &NormalizationConfig,
-) -> ProcessingResult {
+pub fn normalize_event(event: &mut Annotated<Event>, config: &NormalizationConfig) {
     let Annotated(Some(ref mut event), ref mut meta) = event else {
-        return Ok(());
+        return;
     };
 
     let is_renormalize = config.is_renormalize;
 
     if !is_renormalize {
-        normalize(event, meta, config)?;
+        normalize(event, meta, config);
     }
-
-    Ok(())
 }
 
 /// Normalizes the given event based on the given config.
 ///
 /// The returned [`ProcessingResult`] indicates whether the passed event should
 /// be ingested or dropped.
-fn normalize(event: &mut Event, meta: &mut Meta, config: &NormalizationConfig) -> ProcessingResult {
+fn normalize(event: &mut Event, meta: &mut Meta, config: &NormalizationConfig) {
     // Validate and normalize transaction
     // (internally noops for non-transaction events).
     // TODO: Parts of this processor should probably be a filter so we
     // can revert some changes to ProcessingAction)
-    let mut transactions_processor = transactions::TransactionsProcessor::new(
-        config.transaction_name_config.clone(),
-        config.transaction_range.clone(),
-    );
-    transactions_processor.process_event(event, meta, ProcessingState::root())?;
+    let mut transactions_processor =
+        transactions::TransactionsProcessor::new(config.transaction_name_config.clone());
+    let _ = transactions_processor.process_event(event, meta, ProcessingState::root());
 
     // Check for required and non-empty values
     let _ = schema::SchemaProcessor.process_event(event, meta, ProcessingState::root());
```

Comment:
`SchemaProcessor` only modifies the payload to add errors and remarks to `meta`, so running it after validation and timestamp normalization is fine.